client/src/pages/SubmissionContent.vue

```diff
@@ -95,6 +96,28 @@
             </template>
           </q-banner>
           <div v-if="updateMethod == 'upload'">
+            <q-card class="q-pa-md" flat>
+              <q-list>
+                <q-item clickable>
```

Comment:
I don't think this needs to be clickable.
```suggestion
                <q-item>
```